src/components/navbar/MobileNavBar.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import {
+  FaTwitter, FaVimeoV, FaPinterest, FaFacebook, FaGooglePlus,
+} from 'react-icons/fa';
+import Logo from '../img/Logos.png';
+
+const MobileNavBar = () => (
+  // const navigate = useNavigate();
+
+  // const handleClick = () => {
+  //   navigate('/login');
+  //   localStorage.clear();
+  //   window.location.reload();
+  // };
```

Comment:
- [ ] Kindly remove the old commented code present in `MobileNavBar.js and routes/index.js` that way, your repo's cleanliness will be improved. 👍